src/Native/System.Native/pal_networking.h

```diff
@@ -117,6 +117,23 @@ struct LingerOption
     int32_t Seconds; // Number of seconds to linger for
 };
 
+struct IOVector
+{
+    uint8_t* Base;
+    uintptr_t Count;
+};
+
+struct MessageHeader
+{
+    uint8_t* SocketAddress;
```

Comment:
Agree style-wise on uint8_t, but fwiw sizeof(unsigned char) == 1 is guaranteed IIRC. I think the standard allows for bytes/sizeof units to be more than 8 bits, but we'd have other problems in that exotic case.